src/utils/testHelpers.ts

```diff
@@ -3,7 +3,9 @@ import _ from 'lodash';
 import { Dictionary } from '../types';
 import { ResolutionError } from '../index';
 import mockData from '../testData/mockData.json';
+import {default as HttpProvider} from './oldProvider/oldHttpProvider';
 
+export {HttpProvider as oldWeb3Provider};
```

Comment:
1.x may become "old" too in future. We should be more specific, even if it's only our internal code. I suggest we stick to our already existing convention: "Web3Version0"